lib/ruby_routes/router/resource_helpers.rb

```
@@ -0,0 +1,158 @@
+      private
+
+      # Define RESTful routes for a resource.
+      #
+      # @param resource_name [Symbol, String] The name of the resource.
+      # @param options [Hash] Options for customizing the resource routes.
+      #   - `:path` [String] Custom path for the resource.
+      #   - `:controller` [String] Custom controller name.
+      #   - `:nested` [Symbol, String] Name of the nested resource.
+      # @param nested_block [Proc] A block for defining nested routes.
+      # @return [void]
+      def define_resource_routes(resource_name, options = {}, &nested_block)
```

RuboCop (1.76.1)

[warning] Method `RubyRoutes::Router::ResourceHelpers#define_resource_routes` is defined at both lib/ruby_routes/router/resource_helpers.rb:26 and lib/ruby_routes/router/resource_helpers.rb:48.

(Lint/DuplicateMethods)